delta/src/test/scala/ch/epfl/bluebrain/nexus/kg/routes/ResourceRoutesSpec.scala

```diff
@@ -251,8 +259,10 @@ class ResourceRoutesSpec
 
       val endpoints = List(
         s"/v1/resources/${projectMeta.value.organizationUuid}/${projectMeta.uuid}/_/$urlEncodedId",
+        s"/v1/resources/${projectMeta.value.organizationUuid}/${projectMeta.uuid}/$schemaEncodedId/$urlEncodedId",
```

Comment:
please add tests with variations where the schema id segment is just the schema uuid like for example `1988e0be-587e-4a49-a0d0-ea5e63bcc85b` and the resource id segment is just the resource uuid.